utilities/sourceEvents.py

```diff
@@ -514,13 +514,26 @@ def start(targets=None):
     previous_eventId_list = find_all_previous_event_ids(current_app.config['EVENT_COLLECTION'], filter={"dataSourceEventId": {'$exists': 'true'}})
     urls = geturls(targets)
     for url in urls:
+        parsedEvents = list()
+        notShareWithMobileList = list()
         try:
-            rawEvents = download(url)
-            if rawEvents is None:
-                print("Invalid content in: {}".format(url))
+            page_number = 0
```

Comment:
page_number needs to start from 1, right? page_number=0 is the same as providing no page number. Please check this case. Thanks.

Reply:
it seems not. page_number=0 and 1 have different events. Can we confirm from webtool people?
e.g., https://xml.calendars.illinois.edu/eventXML13/7?pageNumber=0

Reply:
Thanks for checking this. It looks like I misread the email conversation with Lance. He said that if no page number is provided, it will default to 0. So, yes, it means that the pageNumber is starting from 0. Sorry for the confusion.